cmd/soci/commands/push.go

```diff
@@ -92,6 +92,9 @@ if they are available in the snapshotter's local content store.
 		}
 
 		indexDesc := indexDescriptors[len(indexDescriptors)-1]
+		if indexDesc.MediaType == soci.OCIArtifactManifestMediaType {
```

Comment:
Why 7:15?

Reply:
Just the first 7 characters of the digest 'sha256:abcdefg...'